### PR TITLE
At cp add badges

### PR DESCRIPTION
**Pull request by atasker**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
-  root 'restaurants#index'
   resources :restaurants
```
Comment (omidbachari) [inline config/routes.rb]: Please use an `only:` to restrict the routes you're not using.

**Comment by omidbachari** [comment]
Implement feedback and merge Mr. @atasker. Just 1-1 me when you merge so I can show you how to do it.

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   devise_for :users
-  root 'restaurants#index'
-  resources :restaurants
+  resources :restaurants, only:
+  [
```
Comment (dpickett) [inline config/routes.rb:4]: all actions are listed here, so it is safe to delete the only directive

Comment (omidbachari) [inline config/routes.rb:4]: Good to know. Note that I advised that they keep it explicit, over their initial inclination to omit the `only:`. They had the right instinct.

**Pull request merged**